src/Raven.Server/Monitoring/Snmp/SnmpWatcher.cs

```diff
@@ -394,6 +412,11 @@ private async Task AddDatabases()
                         LoadDatabase(database, mapping[database]);
                 }
             }
+            catch (Exception e)
+            {
+                if (Logger.IsOperationsEnabled)
+                    Logger.Operations("Failed to update the SNMP mapping", e);
```

Comment:
Can we add database names we tried? - `missingDatabases`

Reply:
done